### PR TITLE
Update instruction

### PR DESCRIPTION
**Pull request by HalynaDumych**
Added info on how to enable cache and set the maximum cache size.

Relates-To: OLPEDGE-2487

Signed-off-by: Halyna Dumych <ext-halyna.dumych@here.com>

### CODE REVIEW

File: docs/create-platform-client-settings.md [hunk 1]
```diff
@@ -34,20 +35,43 @@ You need to create the `OlpClientSettings` object to get catalog and partition m
...
+   retry_settings.retry_condition = [](const olp::client::HttpResponse& response) {
+     return olp::http::HttpStatusCode::TOO_MANY_REQUESTS == response.status;
+   };
    retry_settings.backdown_strategy = ExponentialBackdownStrategy();
    retry_settings.max_attempts = 3;
    retry_settings.timeout = 60;
    retry_settings.initial_backdown_period = 200;
-   };
+   }
+   ;
```
[the remark this answers is not in the document]
Comment (HalynaDumych) [reply docs/create-platform-client-settings.md]: Done.

File: docs/create-platform-client-settings.md [hunk 2]
```diff
@@ -34,20 +35,43 @@ You need to create the `OlpClientSettings` object to get catalog and partition m
...
+
+      > #### Note
+      > By default, the downloaded data is cached in memory, and the maximum size of it is 1 MB.
+
+      ```cpp
+      olp::cache::CacheSettings cache_settings;
+        //On iOS, the path is relative to the Application Data folder.
+        cache_settings.disk_path_mutable = "path to mutable cache";
+        cache_settings.disk_path_protected = "path to protected cache";
+        cache_settings.max_memory_cache_size = 5242880;
```
[the remark this answers is not in the document]
Comment (HalynaDumych) [reply docs/create-platform-client-settings.md]: Changed.

[the remark this answers is not in the document]
Comment (HalynaDumych) [reply docs/create-platform-client-settings.md]: Changed.

File: docs/create-platform-client-settings.md [hunk 3]
```diff
@@ -58,7 +82,7 @@ You need to create the `OlpClientSettings` object to get catalog and partition m
    std::chrono::seconds default_cache_expiration = std::chrono::seconds(200);
```
[the remark this answers is not in the document]
Comment (HalynaDumych) [reply docs/create-platform-client-settings.md]: Combined 6 with 7.

**Review by mykhailo-kuchma — APPROVED** [review]

**Review by andescu — APPROVED** [review]

**Pull request merged**